crates/testing/src/exotest/assertion/mod.rs

```diff
@@ -164,14 +158,14 @@ mod tests {
         .collect();
 
         let runtime = Runtime::new().unwrap();
-        dynamic_assert_using_deno(
-            expected,
-            actual_payload(),
-            "",
-            &testvariables,
-            runtime.handle(),
-        )
-        .unwrap();
+        runtime
```

Comment:
You can just use https://docs.rs/tokio/latest/tokio/attr.test.html#using-the-multi-thread-runtime to avoid `block_on`